resources/migrations/018_add_data_migrations_table.yaml

```diff
@@ -8,7 +8,7 @@ databaseChangeLog:
             columns:
               - column:
                   name: id
-                  type: VARCHAR
+                  type: VARCHAR(254)
                   constraints:
                     primaryKey: true
                     nullable: false
```

Comment:
Adding a [`validCheckSum` list](http://www.liquibase.org/documentation/changeset.html) would be good here or everyone who has already ran this migration is going to have to rollback and run it again.

That's probably not a huge deal, we can just do `lein run migration down-one` or `(db/migration :down-one)` from the REPL. But just a heads up this will break a few of our setups.